OTAnalytics/plugin_ui/nicegui_application.py

```diff
@@ -0,0 +1,75 @@
+from functools import cached_property
+
+from OTAnalytics.plugin_ui.gui_application import OtAnalyticsGuiApplicationStarter
+from OTAnalytics.plugin_ui.nicegui_gui.pages.configuration_bar.container import (
+    ConfigurationBar,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.pages.configuration_bar.project_form import (
+    ProjectForm,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.pages.configuration_bar.svz_metadata_form import (  # noqa
+    SvzMetadataForm,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.pages.visualization_filters import (
+    VisualizationFilters,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.pages.visualization_layers import (
+    VisualizationLayers,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.pages.workspace import Workspace
+
+
+class OtAnalyticsNiceGuiApplicationStarter(OtAnalyticsGuiApplicationStarter):
+
+    def start_ui(self) -> None:
+        from OTAnalytics.plugin_ui.nicegui_gui.endpoints import ENDPOINT_MAIN_PAGE
```

Comment:
Is there a specific reason why these imports are defined here and not at the top of the file?

Reply:
Yes, as with tkinter, we do not want to enforce the installation of ui modules in a cli usage. Hence, this might be a smaller issue for nicegui, because it is a server module.